website/source/docs/agent/checks.html.markdown

```diff
@@ -34,7 +34,10 @@ There are five different kinds of checks:
   with a request timeout equal to the check interval, with a max of 10 seconds.
   It is possible to configure a custom HTTP check timeout value by specifying
   the `timeout` field in the check definition. The output of the check is
-  limited to roughly 4K. Responses larger than this will be truncated.
+  limited to roughly 4K. Responses larger than this will be truncated. HTTP checks
+  also support SSL. By default, a valid SSL certificate is expected. Certificate
+  verification can be turned off by setting the `tls_skip_verify` field to `true`
+  in the check definition.
```

Comment:
Let's also document this here: https://github.com/hashicorp/consul/blob/master/website/source/docs/agent/http/agent.html.markdown#agent_check_register